images/hadoop.yml

```diff
@@ -16,10 +16,21 @@ enabled_repos:
 for_payload: false
 from:
   builder:
-  # Upstream CI builds require a RHEL7 based builder image because it requires protobuf 2.5.0 to compile.
-  # This is not a requirement downstream, so RHEL8 is used. See streams.yml for how this magic is down.
-  - stream: hadoop-builder
+  - member: openshift-base-rhel8
   member: openshift-base-rhel8
+  ci_alignment:
+    # Upstream CI builds require a RHEL7 based builder image because it requires protobuf 2.5.0 for tests.
+    # This is not a requirement downstream; instead RHEL8 must be used.
+    #
+    # Set mirror:true so that DPTP will open FROM alignment PRs upstream, but specify the explicit
+    # upstream images to prevent PR reconciliation from trying to "correct" that difference.
+    mirror: true
```

Comment:
The comment has aged a bit. For example, ART opens the PRs now. `mirror` is not required because there is no `upstream_image` specified.